com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UNetChannel.cs

```diff
@@ -12,11 +14,33 @@ public class UNetChannel
         /// <summary>
         /// The name of the channel
         /// </summary>
-        public NetworkChannel Id;
+        [ReadOnly]
+        public byte Id;
 
         /// <summary>
         /// The type of channel
         /// </summary>
         public QosType Type;
+
+        private class ReadOnlyAttribute : PropertyAttribute { }
+
+        [CustomPropertyDrawer(typeof(ReadOnlyAttribute))]
+        private class ReadOnlyDrawer : PropertyDrawer
```

Comment:
this code lives under `Runtime` — so, I think we should editor things with `#if UNITY_EDITOR` here (also `using UnityEditor;` above as well).